Compression/Compression.cs

```diff
@@ -135,6 +135,32 @@ public static bool ZipData(string zipPath, IEnumerable<KeyValuePair<string, byte
             return success;
         }
 
+        /// <summary>
+        /// Zips the specified byte array into the zipPath
+        /// </summary>
+        /// <param name="zipPath"></param>
+        /// <param name="zipEntry"></param>
+        /// <param name="data"></param>
+        /// <returns></returns>
+        public static bool ZipData(string zipPath, string zipEntry, byte[] data)
```

Comment:
hm think this case is handled by `ZipCreateAppendData` already really, suggest not creating a new method if not required